datajoint/blob.py

```diff
@@ -5,7 +5,7 @@
 
 import zlib
 from itertools import repeat
-import collections
+import collections.abc
```

Comment:
Nm, this has actually been handled correctly already. We can leave as is.

```suggestion
import collections
```